agents/reviewer.md

```
@@ -0,0 +1,106 @@
+
+---
+
+### 🟢 Nice to Have (선택사항)
+> 더 나은 방향에 대한 제안
+
+---
+
+### ✅ 승인 조건
+- [ ] 🔴 Must Fix 항목 모두 수정
+- [ ] 재리뷰 요청 (🔴 항목 수정 후)
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)